### ISSUE TITLE
Prints extra 'DONE' string after running mocha test.

### ISSUE THREAD

**Issue by FerrielMelarpis** [enobufs/mocha-prepare#1]
Hi! It seems that mocha-prepare prints a 'DONE' string at the end of running mocha tests. However, this bugs me a lot since I'm using grunt. Grunt already has its own way of printing information and after finishing all its registered tasks, it prints 'Done.' I would like to have an option where I can customize the message being printed by mocha-prepare. If that's not possible,
maybe adjust the code to use `grunt.log` functionality. Maybe something like this:
```sh
Running "mochaTest:test" (mochaTest) task # This is printed by grunt

Preparing mocha... # This is printed by mocha-prepare

...

Unpreparing mocha... # This is printed by mocha-prepare

Done. # This is printed by grunt
```

**Comment by enobufs** [comment]
It was not really my intention to leave console.log('Done') in the code. Let me simply remove it. If someone would like to print out something like it, he/she can always implement unprepare handler...

**Comment by enobufs** [comment]
I have removed the log, published as v0.0.2.
